devops/arangodb.yml

```diff
@@ -46,6 +46,14 @@
       shell: ARANGODB_DEFAULT_ROOT_PASSWORD={{ arangodb_root_password }} arango-secure-installation
       become: true
 
+    # - name: Ensure there is a read-only user
+    #   executable: /usr/bin/arangosh --server.password {{ arangodb_root_password }}
```

Comment:
Looks like you probably just need more flags when you run this. See [here](https://www.arangodb.com/docs/stable/programs-arangosh-examples.html). The main ones are the server protocol and probably passing in the user. Here's a quote from that doc that seems relevant:

> If the server endpoint is configured for SSL then clients such as arangosh need to connect to it using an SSL socket as well. For example, use http+ssl:// as schema in --server.endpoint for an SSL-secured HTTP connection.

You might also need to append [javascript.execute-string](https://www.arangodb.com/docs/stable/programs-arangosh-options.html#javascript) so it doesn't drop you into a shell at this step.

Reply:
I've done more work locally to try to make this work, which does include extra parameters, but ansible doesn't seem to allow appending arguments to the executable specifier. I'm not sure how to get around this in a sane way.

Reply:
Can you not run is as a shell command like the `arango-secure-installer` above?

Reply:
That command is running a command with the `shell` command, using the default shell (probably `sh` or `bash`). The problem for me isn't the commands themselves, it's that the executable I want to use (`arangosh`) needs a password specified, and ansible recognizes the whole command (with arguments) as the executable, sayings it's not found.

Reply:
With help from @waxlamp, this has been resolved in 45f96ac